proxy/proxy.go

```diff
@@ -244,6 +244,11 @@ type Params struct {
 	// It allows to add additional logic (for example tracing) by providing a wrapper function
 	// which accepts original skipper http.RoundTripper as an argument and returns a wrapped roundtripper
 	CustomHttpRoundTripperWrap func(http.RoundTripper) http.RoundTripper
+
+	// Registry provides key-value API which uses "host:port" string as a key
+	// and returns some metadata about endpoint. Information about the metadata
+	// returned from the registry could be found in routing.metrics interface.
```

Comment:
Is it `routing.metrics`?

Reply:
At the moment its methods are used only in the `routing` Package, so I decided to keep it private until some other package will use some of those methods.